src/main/java/org/opentestsystem/ap/ims/service/ItemService.java

```diff
@@ -35,9 +35,9 @@
 @Component
 public class ItemService {
 
-    private static final boolean NO_DO_NOT_COMMIT_ON_SAVE = false;
+    private static final boolean KEEP_CHANGES_PRIVATE = false;
```

Comment:
I like the rename - the old name was a double negative which was more confusing.